lib/page.js

```diff
@@ -727,6 +726,10 @@ export class Context {
     }
   }
 
+  replaceState() {
```

Comment:
Is this function necessary? I couldn't find where it was being called

Reply:
I really wanted to use it here: https://github.com/Banno/platform-ux/blob/master/projects/banno-online/src/components/bannoweb/login/bannoweb-login-consent.js#L189-L198

It's simply an alias for `context.save()`, but it makes it explicit why you are calling it.